hcp-consul-hcp-vault-ca/confirm.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+export WORKBENCH=$(kubectl get pods -l app=tutorial  -o json | jq -r ".items[0].metadata.name")
+rc =$(echo $?)
```

Comment:
```suggestion
rc=$(echo $?)
```
Space after the variable name generates error